build.rs

```diff
@@ -1,24 +1,37 @@
-use std::path::{Path, PathBuf};
+use std::path::{PathBuf, Path};
 
 fn main() -> std::io::Result<()> {
     println!("cargo:rerun-if-env-changed=LND_REPO_DIR");
-    let lnd_rpc_dir_owned;
-    let dir = match std::env::var_os("LND_REPO_DIR") {
+    let dir= match std::env::var_os("LND_REPO_DIR") {
         Some(lnd_repo_path) => {
             let mut lnd_rpc_dir = PathBuf::from(lnd_repo_path);
             lnd_rpc_dir.push("lnrpc");
-            lnd_rpc_dir_owned = lnd_rpc_dir;
-            &*lnd_rpc_dir_owned
+            lnd_rpc_dir.to_path_buf()
         },
-        None => Path::new("vendor"),
+        None => Path::new("vendor").to_path_buf(),
     };
-
+    
     let lnd_rpc_proto_file = dir.join("lightning.proto");
     println!("cargo:rerun-if-changed={}", lnd_rpc_proto_file.display());
 
+    let protos = [
+        "lightning.proto",
+        "signrpc/signer.proto",
+        "walletkit.proto",
+    ];
```

Comment:
I noticed the directory structure of LND is different. It'd be nice to have it the same so that people can simply point the env var to it and not have to rearrange stuff.

Reply:
fixed